examples/pytorch/pyg/test_interface.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import numpy as np
+import numpy.testing as npt
+import os
+import sys
+import pytest
+import random
+import argparse
+import torch
+
+from examples.pytorch.pyg.train import DeepGNNFeatureStore, DeepGNNGraphStore
+from deepgnn.graph_engine import SamplingStrategy
+from deepgnn.graph_engine.data.citation import Cora
+from torch_geometric.loader import LinkNeighborLoader
+
+
+def test_interface():
```

Comment:
This is not needed, just use an assert in the train.py

Reply:
Train.py uses ppi which is too slow to run in CI pipeline.

Reply:
what stops you from using a smaller graph, which will be also faster to load?

Reply:
We want it to be a useful example that shows actual learning

Reply:
> We want it to be a useful example that shows actual learning. Cora is not big enough.

Why many papers use it for benchmarks?

> Also none of our other examples do testing like this

[Of course we do](https://github.com/microsoft/DeepGNN/blob/1e35f511e6f18857e1aa0c774e9dee23b9adc88c/.github/workflows/ci.yml#L154C9-L155C73). There is no assert in main, we only verify code runs without exceptions and have UT to verify logic. It will be easier to add an assert in this train.py and skip UTs, because you are not adding any new code outside of examples folder.